learn/what_is_meilisearch/comparison_to_alternatives.md

```diff
@@ -75,31 +75,31 @@ Can't find a client you'd like us to support? [Submit your idea or vote for it](
 |   | Meilisearch | Algolia | Typesense | Elasticsearch |
 |---|:---:|:----:|:---:|:---:|
 | Typo tolerant  | ✅ | ✅ | ✅ | 🔶 <br>Needs to be specified by fuzzy queries |
-| Orderable ranking rules | ✅ | ✅ | 🔶 <br>Tie-breaking order is limited by a unique scoring rule | ❌|
-| Custom rules | ✅ | ✅ | 🔶 <br>Limited to one default sorting rule | 🔶 <br>Function score query
+| Orderable ranking rules | ✅ | ✅ | 🔶 <br>Field priority can be changed, but ranking rules order cannot be changed. | ❌|
+| Custom rules | ✅ | ✅ | ✅ | 🔶 <br>Function score query
 | Query field weights | ✅ | ✅ | ✅ | ✅ |
 | Synonyms | ✅ | ✅ | ✅ | ✅ |
 | Stop words | ✅ | ✅ | ❌ | ✅ |
 | Automatic language detection | ✅ | ✅ | ❌ | ❌ |
-| All language supports | ✅ | ✅ | ❌ <br> Only space separated | ✅ |
```

Comment:
Launched in v0.24.0

Reply:
Can we continue [this conversation, here](https://github.com/meilisearch/documentation/pull/2173#issuecomment-1435000867)?

Reply:
I'm actually unsure we want to have the conversation there, as it's a deleted PR now.
Anyway here's where we were

> @Kerollmops
> 
> > I read feedback about a [not-so-great support of Chinese](https://github.com/typesense/typesense/issues/52#issuecomment-1356140139). We work hard at Meilisearch to improve the support of CJK so maybe we can add something about the support of it on the Typesense side, i.e. in progress. What do you think?

>@jasonbosco 
>> 
>> Ah missed seeing that issue. Sure, feel free to mention that it's being improved.
>> 
>> We've been improving support as and when we get feedback from native speakers for different scenarios.

I'm actually fine just leaving check marks to everyone, as @jasonbosco suggested.